src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml.cs

```diff
@@ -189,16 +189,48 @@ internal void SaveWorkspaceAsImage(string path)
             var initialized = false;
             var bounds = new Rect();
 
+            double minX = 0.0, minY = 0.0;
             var dragCanvas = WpfUtilities.ChildOfType<DragCanvas>(this);
             var childrenCount = VisualTreeHelper.GetChildrenCount(dragCanvas);
             for (int index = 0; index < childrenCount; ++index)
             {
                 var child = VisualTreeHelper.GetChild(dragCanvas, index);
                 var firstChild = VisualTreeHelper.GetChild(child, 0);
-                if ((!(firstChild is NodeView)) && (!(firstChild is NoteView)) && (!(firstChild is AnnotationView)))
-                    continue;
 
+                switch (firstChild.GetType().Name)
+                {
+                    case "NodeView":
+                    case "NoteView":
+                    case "AnnotationView":
+                        break;
```

Comment:
they are followed by a `default : continue` statement, but `break` is for `switch` and `continue` is for `for` loop. sort of context switch..